File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/PHY2001U04CboJubsuGubunInfo.java
package nta.med.data.model.ihis.phys;

import java.io.Serializable;

public class PHY2001U04CboJubsuGubunInfo implements Serializable {
	private String code        ;
	private String codeName   ;
	private String sortKey    ;
	public PHY2001U04CboJubsuGubunInfo(String code, String codeName,
			String sortKey) {
		super();
		this.code = code;
		this.codeName = codeName;
		this.sortKey = sortKey;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getSortKey() {
		return sortKey;
	}
	public void setSortKey(String sortKey) {
		this.sortKey = sortKey;
	}
	

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY2001U04CboJubsuGubunInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
    public class PHY2001U04CboJubsuGubunInfo
    {
        private String _code;
        private String _codeName;
        private String _sortKey;

        public String Code
        {
            get { return this._code; }
            set { this._code = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public String SortKey
        {
            get { return this._sortKey; }
            set { this._sortKey = value; }
        }

        public PHY2001U04CboJubsuGubunInfo() { }

        public PHY2001U04CboJubsuGubunInfo(String code, String codeName, String sortKey)
        {
            this._code = code;
            this._codeName = codeName;
            this._sortKey = sortKey;
        }

    }
}